src/style.css

```diff
@@ -0,0 +1,60 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star {
+  background-image: url(./images/star.svg);
+}
+
+.stars--5 .stars__star {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(1) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(n+2) {
+  background-image: url(./images/star.svg);
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(./images/star-active.svg);
+}
+
+.stars--2 .stars__star:nth-child(n+3) {
+  background-image: url(./images/star.svg);
```

Comment:
add not active star do default `.stars__star`
and then you will change it to active for nth'childs

Reply:
optimised this. hope it is better now. thank you for recomendations!